dashboard/app/controllers/pd/application_dashboard_controller.rb

```diff
@@ -6,12 +6,27 @@ def index
       view_options(full_width: true)
 
       is_workshop_admin = current_user.permission? UserPermission::WORKSHOP_ADMIN
+      workshops = if is_workshop_admin
+        Pd::Workshop.where(subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP)
+      else
+        Pd::Workshop.where(subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP, regional_partner: current_user.regional_partners)
+      end.in_state(Pd::Workshop::STATE_NOT_STARTED).scheduled_start_on_or_before(Date.today.end_of_year)
```

Comment:
What about course? Don't we want this to be for CSD or CSP (whichever is relevant to the application) rather than both?

Reply:
Hmm - I don't think we want to filter out course here but we should include course in the object we return. We don't know the application until the user is at the detail page

Reply:
I'm going to make this change in the other branch

Reply:
At least split up the workshops by course, something like
```
{
  csd: [...],
  csp: [...]
}
```

I wonder if we should be returning these from an AJAX call instead of serializing with the dashboard js. Probably not a big deal - this is fine - but we definitely want to split by course.